src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py

```diff
@@ -75,21 +82,28 @@ class (the rest are helpers)
         self.process_organization()
         self.process_project()
 
-        # Set up the default names for the rest of the things we need to create
-        self.repository_name = self.project_name
-        self.service_endpoint_name = self.organization_name + self.project_name
-        self.build_definition_name = self.project_name + " INITIAL AZ CLI BUILD"
-        self.release_definition_name = self.project_name + " INITIAL AZ CLI RELEASE"
-
+        # Generate Azure pipenline build yaml
         self.process_yaml()
-        self.process_repository()
 
+        # Allow user to choose the uploading destination
+        self.process_local_repository()
+        self.process_remote_repository()
+
+        # Set up the default names for the rest of the things we need to create
         self.process_service_endpoint()
         self.process_extensions()
 
+        # Start build process and release artifacts to azure functions app
+        self.process_build_and_release_definition_name()
         self.process_build()
         self.process_release()
 
+        # Advise user to reuse the pipeline build and release by pushing to remote
+        print()
```

Comment:
Suggest use `logger.warning()`. `print` is generally discouraged for lack of lots of advantage of using python logging, also printed text goes to `stdout`, and for consistency, should only contain data, not status/error.
For information you don't like users to see by default, use `logger.info()` or `logger.debug()`

Reply:
Thanks @yugangw-msft, I've changed all print statement to logger statements.